docs/csharp/programming-guide/classes-and-structs/named-and-optional-arguments.md

```diff
@@ -102,7 +102,7 @@ Optional parameters in ExampleMethod
 ## COM Interfaces  
  Named and optional arguments, along with support for dynamic objects and other enhancements, greatly improve interoperability with COM APIs, such as Office Automation APIs.  
   
- For example, the [AutoFormat](http://go.microsoft.com/fwlink/?LinkId=148201) method in the Microsoft Office Excel [Range](http://go.microsoft.com/fwlink/?LinkId=148196) interface has seven parameters, all of which are optional. These parameters are shown in the following illustration.  
+ For example, the [AutoFormat](https://msdn.microsoft.com/library/microsoft.office.interop.excel.range.autoformat(office.11).aspx) method in the Microsoft Office Excel [Range](https://msdn.microsoft.com/library/microsoft.office.interop.excel.range(VS.80).aspx) interface has seven parameters, all of which are optional. These parameters are shown in the following illustration.  
```

Comment:
my recommendation would be to point to the latest version of the AutoFormat API:
https://msdn.microsoft.com/library/microsoft.office.interop.excel.range.autoformat(v=office.15).aspx

same for range:
https://msdn.microsoft.com/library/microsoft.office.interop.excel.range(v=vs.15).aspx

Reply:
Sure thing, quick question though. What is the difference between v=office.15 and v=vs.15. For range, it looks like they go to the same page. If there is no significant difference, does it make sense to use v=office.15 for both links for consistency?